pkg/controller/pod/pod_controller.go

```diff
@@ -89,5 +89,15 @@ func (r *ReconcilePolicy) Reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{}, err
 	}
 
+	// allocate only when the pod is ready
```

Comment:
how is this change related?

Reply:
we need this change.
run the allocation after the pod is created (have a name and not only a generateName object) this will update the value on the mac address map to true and create the pod entry to in `podToMacPoolMap`